perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/MailManagerImpl.java

```diff
@@ -1729,6 +1729,7 @@ private void mailValidation(Application app, ApplicationMail mail, List<Applicat
 									newValue += "?vo="+ getUrlEncodedString(app.getVo().getShortName());
 									newValue += ((app.getGroup() != null) ? "&group="+ getUrlEncodedString(app.getGroup().getName()) : EMPTY_STRING);
 									newValue += "&i=" + URLEncoder.encode(i, StandardCharsets.UTF_8) + "&m=" + URLEncoder.encode(m, StandardCharsets.UTF_8);
+									newValue += "&target=" + BeansUtils.stringToMapOfAttributes(app.getFedInfo()).get("redirectURL");
```

Comment:
We probably shouldn't add the target if the redirectURL is not filled.

Reply:
Fixed, thanks.